cli/commands/db.go

```diff
@@ -67,7 +67,7 @@ func newDBCommand(c *cli.Container) *cobra.Command {
 			c.Tracker.Infof("starting %s...", cliName)
 
 			err = command.New(command.WithStdin(os.Stdin), command.WithStdout(os.Stdout), command.WithStderr(os.Stderr)).
-				Exec(cliName, strings.Fields(connArg)...)
+				Exec(cmd.Context(), cliName, strings.Fields(connArg)...)
```

Comment:
question: does `Context()` just return the cmd name?

Reply:
It returns a `context.Context`. I recommend reading the package docs: https://pkg.go.dev/context

It's used for cancelations/deadlines, in this case if the context gets cancelled then the subprocess spawned by Exec will be killed. E.g. if the user does ctrl-C, it will gracefully kill the subprocess.

Reply:
aight cool, thanks for clarifying